tfhe/src/shortint/server_key/tests/shortint.rs

```diff
@@ -532,7 +532,7 @@ where
 {
     let keys = KEY_CACHE.get_from_param(param);
     let (cks, sks) = (keys.client_key(), keys.server_key());
-    let double = |x| 2 * x;
+    let double = |x| (2 * x) % sks.message_modulus.0 as u64;
```

Comment:
was this test failing ?
otherwise I agree that this change could destroy the padding bit

Reply:
With the new `assert!(f_eval < modulus_sup as u64)` the `shortint_generate_lookup_table` test was failing with all the parameters I think